src/kibana/components/doc_table/components/table_row/cell.html

```diff
@@ -1,3 +1,3 @@
-<td <%= timefield ? 'class="discover-table-timefield"  width="1%"' : '' %>>
+<td <%= timefield ? 'class="discover-table-timefield"  width="1%"' : 'class="discover-table-datafield"' %>>
```

Comment:
Maybe we could add another case here, we have a special class and style for the timefield I think we could use one for _source as well. I'm not a fan of the way that it looks when whitespaced: 
![image](https://cloud.githubusercontent.com/assets/1329312/7994441/551729a2-0ac2-11e5-97e0-b586dd62cb62.png)